docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironmentTest.java

```diff
@@ -46,11 +56,27 @@
     @Mock
     Event<PropertyReportEvent> propertyReportEvent;
 
+    @Mock
+    private Cube cube;
+
     @Captor
     ArgumentCaptor<PropertyReportEvent> propertyReportEventArgumentCaptor;
 
+    private CubeRegistry cubeRegistry;
+
+    private Statistics statistics;
+
     @Before
-    public void configureDockerExecutor() {
+    public void configureDockerExecutorAndCubeRegistry(){
+        configureDockerExecutor();
+        try {
+            configureCube();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Just make this method throwing exception, otherwise it will continue executing tests, where we want to fail fast.